scripts/images.sh

```diff
@@ -14,6 +14,6 @@ image=${DOCKER_REGISTRY}/dispatch-${PACKAGE}:${TAG}
 echo $image
 
 docker build -t $image -f images/${PACKAGE}/Dockerfile .
-if [ -z $NO_PUSH ]; then
+if [ -z $NO_PUSH ] && [ -n "$PUSH_IMAGES" ]; then
```

Comment:
Seems like NO_PUSH is not needed anymore?

Reply:
I'm gonna remove it, if it's not used by anyone.

Reply:
I think it's safe to remove, because with your change this variable has no effect anymore